web/src/components/users/FirstUser.test.jsx

```diff
@@ -86,6 +88,20 @@ it("allows defining a new user", async () => {
   });
 });
 
+it("doest not allow to confirm the settings if the user name and the password are not provided", async () => {
+  const { user } = installerRender(<FirstUser />);
+  const firstUser = await screen.findByText(/A user/);
+  const button = within(firstUser).getByRole("button", { name: "is not defined" });
+  await user.click(button);
+
+  await screen.findByRole("dialog");
```

Comment:
I would be consistent with the `within` usage above. See https://github.com/yast/d-installer/blob/27cfff5f127945629d8ed65509584d74a6739d3f/web/src/components/core/About.test.jsx#L36-L38.

It is not that the current solution does not work, but I would say it is more correct (and less fragile) to make sure that you interact with the dialog.